docs/docs/styling.md

```diff
@@ -1,14 +1,14 @@
 ---
-title: Styling
+title: Estilizando
 overview: true
 ---
 
-There are many ways to style your website. They can roughly be grouped into three styling approaches:
+Há várias formas de estilizar o seu _website_. Eles podem ser agrupadas em três abordagens de estilização:
 
-- [**Global CSS Files**](/docs/global-css/): the traditional way to style a website. CSS rules are declared globally and styles are applied depending on specificity and inheritance.
-- [**Modular Stylesheets**](/docs/css-modules): CSS rules are written traditionally but consumed with JavaScript and scoped locally to avoid unintended side-effects elsewhere. Works out-of-the-box with Gatsby.
-- [**CSS-in-JS**](/docs/css-in-js/): locally-scoped CSS written and consumed in JavaScript, enabling the easier use of dynamic styling and other features. Requires the use of third-party libraries.
+- [**Arquivos CSS Globais**](/docs/global-css/): a forma tradicional de estilizar um _website_. Regras CSS são declaradas globalmente e os estilos são aplicados dependendo da especificidade e herança entre os elementos.
+- [**Folhas de Estilo Modulares**](/docs/css-modules): regras CSS são escritas tradicionalmente (através de arquivos CSS), mas são consumidas por JavaScript e possuem escopo definido localmente para evitar efeitos colaterais indesejados em outros lugares. Funciona imediatamente com Gatsby.
```

Comment:
```suggestion
- [**Folhas de Estilo Modulares**](/docs/css-modules): regras CSS são escritas tradicionalmente (através de arquivos CSS), mas são consumidas por JavaScript e possuem escopo definido localmente para evitar efeitos colaterais indesejados em outros lugares. Funciona imediatamente com Gatsby.
```
Creio que "Works out-of-the-box with Gatsby" não seja exatamente isso, mas não sei ao certo qual a tradução. Vou ir atrás e quando encontrar, eu comento por aqui.

Reply:
Algumas ideias de tradução:
- Funciona muito bem com o Gatsby.
- Pronto para usar com o Gatsby.
- O Gatsby já vem com esta funcionalidade.
- Vem pronto para o uso no Gatsby.
- Pré-configurado no Gatsby